tests/oci_test.c

```diff
@@ -1139,6 +1141,57 @@ START_TEST(test_cc_oci_toggle) {
 	cc_oci_config_free (config);
 } END_TEST
 
+START_TEST(test_cc_oci_create_cgroup_files) {
+	struct cc_oci_config* config = cc_oci_config_create();
+	GPid workload_pid = 1000;
+	g_autofree gchar *tmpdir, *tasks, *procs;
```

Comment:
Nit: these should really be set to `NULL` for safety. However, they are all assigned to so there is no actual problem here.